applications/tari_console_wallet/src/ui/components/network_tab.rs

```diff
@@ -45,48 +46,49 @@ impl NetworkTab {
             .add_column(Some("User Agent"), Some(MAX_WIDTH.saturating_sub(93)), column2_items);
         column_list.render(f, list_areas[0], &mut ListState::default());
     }
-}
 
-impl<B: Backend> Component<B> for NetworkTab {
-    fn draw(&mut self, f: &mut Frame<B>, area: Rect, app_state: &AppState) {
-        // This is all dummy content and layout for review
-        let main_chunks = Layout::default()
-            .constraints([Constraint::Length(1), Constraint::Length(8), Constraint::Min(10)].as_ref())
-            .split(area);
+    pub fn draw_base_node_peer<B>(&self, f: &mut Frame<B>, area: Rect, app_state: &AppState)
+    where B: Backend {
+        let block = Block::default().borders(Borders::ALL).title(Span::styled(
+            "Base Node Peer",
+            Style::default().fg(Color::White).add_modifier(Modifier::BOLD),
+        ));
+        f.render_widget(block, area);
+
+        let base_node = app_state.get_base_node_peer();
+        let public_key = base_node.public_key.to_hex();
+        let public_address = base_node.addresses.first().unwrap().to_string();
```

Comment:
This unwrap is risky, I would handle the case that for some reason there isn't a first address here and just default to a blank string rather than panic using unwrap_or_default(...)